backend/btrixcloud/crawlconfigs.py

```diff
@@ -359,12 +365,17 @@ async def update_crawl_config(self, cid: uuid.UUID, update: UpdateCrawlConfig):
 
         return {"success": True}
 
-    async def get_crawl_configs(self, archive: Archive):
+    async def get_crawl_configs(self, archive: Archive, tags: List[str] = None):
```

Comment:
should this be `..., tags: Union[List[str], None] = None`? or `Optional[List[str]] = None` I suppose